qemu/tests/qemu_guest_agent.py

```diff
@@ -937,12 +923,15 @@ def setup(self, test, params, env):
             self.vm = vm
             session = self._get_session(params, self.vm)
 
+            qemu_ga_pkg_path = self.get_qga_pkg_path(session, params, self.vm)
+            self.gagent_install_cmd = params.get("gagent_install_cmd") % qemu_ga_pkg_path
```

Comment:
Is it possible to put `gagent_install_cmd = "start /wait %s /quiet" % self.qemu_ga_pkg_path or qemu_ga_pkg_path` directly into  `gagent_install`?
and same for the uninstall.